src/Blueprint/Blueprint/Models/PSManagedServiceIdentity.cs

```diff
@@ -0,0 +1,29 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.Blueprint.Models
+{
+    public class PSManagedServiceIdentity
+    {
+        public string Type { get; set; }
+        public string PrincipalId { get; set; }
+        public string TenantId { get; set; }
+    }
```

Comment:
This needs to handle user-assigned identities

Reply:
Adding.

---
In reply to: [257122019](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257122019) [](ancestors = 257122019)

Reply:
Actually adding this as one of the tasks for next phase.

---
In reply to: [257333104](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257333104) [](ancestors = 257333104,257122019)